homeassistant/helpers/entity_registry.py

```diff
@@ -161,7 +161,11 @@ def __init__(self, hass: HomeAssistant) -> None:
 
     @callback
     def async_get_device_class_lookup(self, domain_device_classes: set) -> dict:
```

Comment:
Should we improve typing here as well?

```py
    def async_get_device_class_lookup(self, domain_device_classes: set[tuple[str, str]])) -> dict:
```

Although, I guess the device class may be `None` as well, in that case:

```py
    def async_get_device_class_lookup(self, domain_device_classes: set[tuple[str, str | None]]) -> dict:
```